locale/fr.json

```diff
@@ -4,20 +4,20 @@
         "humanizer_language": "fr"
     },
     "restarter": {
-        "start_timeout": "failed to start in time",
+        "start_timeout": "Impossible de démarrer le serveur dans les temps",
         "crash_detected": "le serveur a planté",
-        "hang_detected": "hang detected",
-        "partial_hang_warn": "Due to a partial hang, this server will restart in 1 minute. Please disconnect now.",
-        "partial_hang_warn_discord": "Due to a partial hang, **%{servername}** will restart in 1 minute.",
+        "hang_detected": "Blocage detecté",
+        "partial_hang_warn": "En raison d'un blocage partiel, le serveur redémarrera dans 1 minute. Veuillez vous déconnecter maintenant.",
+        "partial_hang_warn_discord": "En raison d'un blocage partiel, **%{servername}** redémarrera dans 1 minute.",
         "schedule_reason": "redémarage prévu à %{time}",
         "schedule_warn": "Le serveur redémarrera dans %{smart_count} minute. Merci de vous déconnecter. |||| Le serveur redémarrera dans %{smart_count} minutes.",
         "schedule_warn_discord": "**%{servername}** redémarrera dans %{smart_count} minute. |||| **%{servername}** redémarrera dans %{smart_count} minutes."
     },
     "ban_messages": {
-        "kick_temporary": "(%{author}) You have been banned from this server for \"%{reason}\". Your ban will expire in: %{expiration}.",
-        "kick_permanent": "(%{author}) You have been permanently banned from this server for \"%{reason}\".",
-        "reject_temporary": "You have been banned from this server.\nYour ban will expire in: %{expiration}.\nBan ID: %{id}.\nBan Reason: %{reason}.\nBanned by: %{author}.",
-        "reject_permanent": "You have been permanently banned from this server.\nBan ID: %{id}.\nBan Reason: %{reason}.\nBanned by: %{author}."
+        "kick_temporary": "(%{author}) Vous avez été banni pour: \"%{reason}\". Votre bannissement expirera dans: %{expiration}.",
+        "kick_permanent": "(%{author}) Vous avez été définitivement banni de ce serveur pour: \"%{reason}\".",
+        "reject_temporary": "Vous avez été banni de ce serveur.\nVotre bannissement expirera dans: %{expiration}.\nBan ID: %{id}.\nRaison du Ban: %{reason}.\nBanni par: %{author}.",
+        "reject_permanent": "Vous avez été définitivement banni de ce serveur pour.\nBan ID: %{id}.\nRaison du Ban: %{reason}.\nBanni par: %{author}."
```

Comment:
The word (`pour`) must be removed.

Reply:
I'll make the change this night.
Thank you